packages/atomic/cypress/integration/segmented-facet-scrollable-actions.ts

```diff
@@ -0,0 +1,42 @@
+import {
+  TagProps,
+  TestFixture,
+  generateComponentHTML,
+  addElement,
+} from '../fixtures/test-fixture';
+import {segmentedFacetComponent} from './facets/segmented-facet/segmented-facet-selectors';
+import {
+  scrollableComponent,
+  ScrollableSelectors,
+} from './segmented-facet-scrollable-selectors';
+
+type direction = 'right' | 'left';
+
+let scrollable!: HTMLElement;
+
+function appendSegmentedFacet(facetProps: TagProps) {
+  scrollable.append(
+    generateComponentHTML('atomic-segmented-facet', facetProps)
+  );
+}
+
+export const addScrollable =
+  (facetProps: TagProps = {}) =>
+  (env: TestFixture) => {
+    scrollable = generateComponentHTML('atomic-segmented-facet-scrollable');
+    appendSegmentedFacet(facetProps);
+    addElement(env, scrollable);
+  };
+
+export function clickArrow(direction: direction) {
+  direction === 'left'
+    ? ScrollableSelectors.leftArrow().click()
+    : ScrollableSelectors.rightArrow().click();
+}
+
+export function scroll(direction: direction, msDuration: number) {
+  ScrollableSelectors.horizontalScroll().scrollTo(direction, {
+    duration: msDuration,
+  });
+  cy.get(scrollableComponent).trigger('wheel');
```

Comment:
Does not trigger the event.

Reply:
Alternatively, you could do `cy.get(scrollableComponent).then(([el]) => el.dispatchEvent(...))`. Although ideally, it would be nicer if the cypress method worked 🤔

Reply:
Will try this !